### PR TITLE
fix: remove Showing... count message

### LINKED ISSUE #80: Downloads page: table expansion descriptions
The description adjacent to the "show older releases" button isn't quite right because the number we quote for the total available entries is for individual build artifacts, not releases (release is, e.g. v1.10.26). Either change the wording from "releases" to "build artifacts" or just remove and only have "load more" button.

From Peter:

"Also the line Showing `20 latest releases of a total 1879 releases `is a bit wonky, because there are not 1879 releases, but perhaps build artifacts. I'm not sure this is really needed, maybe just having the button there to load more releases is enough.

### PR DESCRIPTION
**Pull request by nhsz**
This PR fixes #80 by removing the number of releases displayed count message

### CODE REVIEW

File: src/pages/downloads.tsx [hunk 1]
```diff
@@ -374,25 +374,13 @@ const DownloadsPage: NextPage<Props> = ({ data }) => {
...
-                    a total ${totalStableReleases} releases`
-                    : `No releases`}
-                </Text>
-              </Stack>
-
               {totalStableReleases > amountStableReleases && (
                 <Stack
                   sx={{ mt: '0 !important' }}
-                  borderLeft={{ base: 'none', md: '2px solid #11866f' }}
+                  borderLeft={{ base: 'none', md: '2px solid #06fece' }}
```
Comment (wackerow) [inline src/pages/downloads.tsx:383]:
```suggestion
                  borderLeft={{ base: 'none', md: '2px solid var(--chakra-colors-primary)' }}
```

File: src/pages/downloads.tsx [hunk 2]
```diff
@@ -440,22 +428,13 @@ const DownloadsPage: NextPage<Props> = ({ data }) => {
...
-                    a total ${totalDevBuilds} releases`
-                    : `No releases`}
-                </Text>
-              </Stack>
-
               {totalDevBuilds > amountDevBuilds && (
                 <Stack
                   sx={{ mt: '0 !important' }}
-                  borderLeft={{ base: 'none', md: '2px solid #11866f' }}
+                  borderLeft={{ base: 'none', md: '2px solid #06fece' }}
```
Comment (wackerow) [inline src/pages/downloads.tsx:437]:
```suggestion
                  borderLeft={{ base: 'none', md: '2px solid var(--chakra-colors-green-200)' }}
```

Comment (wackerow) [reply src/pages/downloads.tsx:437]: Actually:
image

This needs to be light/dark mode responsive... we should be using `primary` here
```suggestion
                  borderLeft={{ base: 'none', md: '2px solid var(--chakra-colors-primary)' }}
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Left a small fix to use our chakra theme variables to make the border color light/dark mode responsive. Then should be good to go!

**Comment by nhsz** [comment]
> https://deploy-preview-118--melodious-puffpuff-8e1109.netlify.app

Thx, updated with the suggestions 👌

**Review by wackerow — APPROVED** [review]

**Pull request merged**